charts/opensrp-web/values.yaml

```diff
@@ -38,7 +38,9 @@ service:
 ingress:
   enabled: false
   className: ""
-  annotations: {}
+  annotations:
+    nginx.ingress.kubernetes.io/configuration-snippet: |
```

Comment:
Suggest you document this and remove it from default ingress configuration. This will not work if another ingress controller other than nginx is used.

Reply:
ok, yeah makes sense. though that also means we will have to duplicate this in every deployment file. I think I will move this to the express server code, seems like the next best choice.